index.html

```diff
@@ -63,7 +63,212 @@ <h1 class="header">I'm Lawrence Glad to see you</h1>
+                    required.</div>
+                <div class="tags2">
+                    <p class="text11">html</p>
+                    <p class="text22">css</p>
+                    <p class="text33">javascript</p>
+                </div>
+                <div class="action2">
+                    <div class="see2">
+                        <div class="see-it2">
+                            <h2> See Project</h2>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <div class="container4">
+            <div class="right-block2">
+
+            </div>
+            <div class="left-block3">
+                <div class="frame2">
+                    <div class="title2">Tonic</div>
+                    <div class="sub2">
+                        <p class="can2">CANOPY</p>
+                        <i class="list11"> Back End Dev</i>
+                        <i class="list22"> 2015 </i>
+
+                    </div>
+                </div>
+                <div class="primary-text2">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.</div>
+                <div class="tags2">
+                    <p class="text11">html</p>
+                    <p class="text22">css</p>
+                    <p class="text33">javascript</p>
+                </div>
+                <div class="action2">
+                    <div class="see2">
+                        <div class="see-it2">
+                            <h2> See Project</h2>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+        <div class="container5">
+            <div class="right-block3">
+
+            </div>
+            <div class="left-block3">
+                <div class="frame2">
+                    <div class="title2">Multi-Post Stories</div>
+                    <div class="sub2">
+                        <p class="can2">CANOPY</p>
+                        <i class="list11"> Back End Dev</i>
+                        <i class="list22"> 2015 </i>
+
+                    </div>
+                </div>
+                <div class="primary-text2">A daily selection of privately personalized reads; no accounts or
+                    sign-ups
+                    required.</div>
+                <div class="tags2">
+                    <p class="text11">html</p>
+                    <p class="text22">css</p>
+                    <p class="text33">javascript</p>
+                </div>
+                <div class="action2">
+                    <div class="see2">
+                        <div class="see-it2">
+                            <h2> See Project</h2>
+                        </div>
+                    </div>
+                </div>
+
+            </div>
+        </div>
+        <div class="container6">
```

Comment:
- [ ] The same case here,Kindly wrap your about section in a `section` tag for semantic and accessibility reasons.
- [ ]  Adjust the spacing of elements on your about section and style them to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/192956379-83437d7a-fa67-4a81-ba02-0750682752c0.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/192956501-4e31fe60-c362-465e-9d69-9b1801fa35d8.png)

- [ ] Kindly provide your each of your languages a background to match the Figma design

![image](https://user-images.githubusercontent.com/98453979/192956960-80d722fa-d4ef-4071-9be7-884cb9265b97.png)

### Checkout figma

![image](https://user-images.githubusercontent.com/98453979/192957119-3c33f798-07de-42be-ac3c-82973023ded8.png)